src/admin-views/aggregator/settings.php

```diff
@@ -5,7 +5,7 @@
  */
 $internal = array();
 $use_global_settings_phrase = esc_html__( 'Use global import settings', 'the-events-calendar' );
-$post_statuses = get_post_statuses( array() );
+$post_statuses = get_post_statuses();
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 14 spaces but found 1 space

<sub>hash: 17ad3970ecaab2cec118a465303c3247</sub>